### PR TITLE
Transformation Event

### PR DESCRIPTION
**Pull request by vivienfan**
This PR adds yml for transformation event (a subevent extended from `Event`).

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: docs/openapi/components/schemas/common/TransformEvent.yml [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+          "type":[
+            "Organization"
+          ]
+        }
+      }
+    ],
+    "newProducts":[
+      "https://api-staging.fabricator.metaltrail.com/v1/credentials/verify/5c93de07-5d94-4357-85cf-aefc9ccbff97"
+    ],
+    "consumedProducts":[
```
Comment (OR13) [inline docs/openapi/components/schemas/common/TransformEvent.yml:198]: might be good to chat a bit about these URLS

File: docs/openapi/components/schemas/common/TransformEvent.yml [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    title: Place
+    description: Information on where the event took place.
+    $ref: ./Place.yml
+    $linkedData:
+      term: place
+      '@id': https://schema.org/Place
+  actor:
+    title: Event Actors
+    description: >-
+      List of organizations acted on the event, https://w3id.org/traceability#Organization
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/TransformEvent.yml:38]:
```suggestion
      List of organizations which acted during the event, https://w3id.org/traceability#Organization
```

File: docs/openapi/components/schemas/common/TransformEvent.yml [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+          "Product"
+        ],
+        "weight":{
+          "type":[
+            "MeasuredValue"
+          ],
+          "value":"3",
+          "unitCode":"Tonne"
+        },
+        "description":"Bars/rods, i/nas,hr, in irreg wnd coils of circ cross section, <14mm in dia",
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/TransformEvent.yml:123]:
```suggestion
        "description":"Bars/rods, i/nas, hr, in irreg wnd coils of circ cross section, <14mm in dia",
```

File: docs/openapi/components/schemas/common/TransformEvent.yml [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+      - type: string
+        enum:
+          - Event
+          - TransformEvent
+  eventType:
```
Comment (nissimsan) [inline docs/openapi/components/schemas/common/TransformEvent.yml:15]: Am I interpreting this correctly that these are all types of steel transformations?

File: docs/openapi/components/schemas/common/TransformEvent.yml [hunk 5]
```diff
@@ -0,0 +1,203 @@
+$linkedData:
+  term: TransformEvent
```
Comment (nissimsan) [inline docs/openapi/components/schemas/common/TransformEvent.yml:2]: If this is meant as steel-specific, should that be reflected in the definition? A `SteelTransformEvent` could be a type of a more generic `SupplyChainValueAdd` event.

**Review by nissimsan — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

File: docs/openapi/components/schemas/common/TransformEvent.yml (hunk 5 above)
Comment (vivienfan) [reply docs/openapi/components/schemas/common/TransformEvent.yml:2]: should be a generic one, not steel specific. enums for eventType have be removed.

File: docs/openapi/components/schemas/common/TransformEvent.yml (hunk 1 above)
Comment (vivienfan) [reply docs/openapi/components/schemas/common/TransformEvent.yml:272]: updated

File: docs/openapi/components/schemas/common/TransformEvent.yml (hunk 4 above)
Comment (vivienfan) [reply docs/openapi/components/schemas/common/TransformEvent.yml:15]: this transform event should be a generic one for different commodities, not steel specific. enums for eventType have be removed.

**Pull request merged**